src/features/wallet/SideBarMenu.tsx

```
@@ -24,19 +30,73 @@ export function SideBarMenu({
+          if (addrInfo.address) addresses.push(addrInfo.address);
+        });
+      }
+    });
+    return addresses;
+  }, [accounts]);
+
+  // Get all warp route addresses from configured routes
+  const warpRouteAddresses = useMemo(() => {
+    const addresses: string[] = [];
+    Object.values(routerAddressesByChainMap).forEach((addressSet) => {
+      addressSet.forEach((addr) => addresses.push(addr));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)